conf/uvek.conf.part

```diff
@@ -1506,6 +1506,24 @@ source src_ch_bfe_komo_projekte : def_searchable_features
          from bfe.komo_projekte
  }
 
+source src_ch_bfe_thermische_netze : def_searchable_features
+ {
+     sql_db = uvek_dev
+     sql_query = \
+         SELECT xtf_id as id \
+             , name as label \
+             , 'feature' as origin \
+             , remove_accents(concat_ws(' ',name)) as  detail \
```

Comment:
if you only have one attribute in the detail, you can use it like this:
```SQL
            , remove_accents(name) as  detail 
```

Reply:
Thanks for the review. 

I just checked the data in xtf file and I can confirm that there are exactly 1000 entries.